internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -284,6 +281,47 @@ func (opts *Opts) askClusterOptions() error {
 	return nil
 }
 
+func (opts *Opts) defaultRegions() ([]string, error) {
+	cloudProviders, err := opts.store.CloudProviderRegions(opts.ConfigProjectID(), opts.Provider, tier, false)
+
+	if err != nil {
+		return nil, err
+	}
+
+	if len(cloudProviders.Results) == 0 || len(cloudProviders.Results[0].InstanceSizes) == 0 {
+		return nil, fmt.Errorf("no regions available")
```

Comment:
[nit] if you don't need interpolation just use `errors.New`
```suggestion
		return nil, errors.New("no regions available")
```